azure-go/go.mod

```diff
@@ -3,6 +3,6 @@ module ${PROJECT}
 go 1.18
 
 require (
-	github.com/pulumi/pulumi-azure-native/sdk v1.93.0
-	github.com/pulumi/pulumi/sdk/v3 v3.54.0
+	github.com/pulumi/pulumi-azure-native/sdk arm2pulumi-v1.95.0
```

Comment:
Not sure how this got here -- probably something timing related. It's not correct, though, so I'll re-run the script to generate new values for this and whatever others have changed since this was submitted.